packages/react-vapor/src/components/table-hoc/examples/TableHOCServerExamples.tsx

```diff
@@ -144,7 +143,7 @@ const tableDatePickerConfig = {
     ],
 };
 
-const mapStateToProps = (state: IReactVaporTestState) => ({
+const mapStateToProps = (state: any) => ({

```

Comment:
wanted changes ?

Reply:
yep, `IReactVaporTestState` is declared in `/src/utils/tests/TestUtils.tsx` which should only be used by unit tests. So by importing it in the examples, it messes up with the type checking and the declarations generation.